tests/integration/privateweave/kubernetes.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_autoscaling_group" "bastion-privateweave-example-com" {
   launch_configuration = "${aws_launch_configuration.bastion-privateweave-example-com.id}"
   max_size = 1
   min_size = 1
-  vpc_zone_identifier = ["${aws_subnet.private-us-test-1a-privateweave-example-com.id}"]
+  vpc_zone_identifier = ["${aws_subnet.utility-us-test-1a-privateweave-example-com.id}"]
```

Comment:
The utility subnet (ELBs, NAT gateways) is only used in private topologies, and is always public.  I did debate renaming it to `dmz` (as I _think_ that's the common term) but utility is probably at least as clear and less likely to offend.